components/knowledge/FileModal.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  useDisclosure,
+} from '@nextui-org/react';
+import { Image } from '@nextui-org/image';
+import { BiPlus } from 'react-icons/bi';
+import { NotionFileModal } from '@/components/knowledge/Notion';
+import { isNotionConfigured, runNotionSync } from '@/actions/knowledge/notion';
+import { useState } from 'react';
+
+interface FileModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  handleAddFile: () => void;
+}
+
+const FileModal = ({ isOpen, onClose, handleAddFile }: FileModalProps) => {
+  const notionModal = useDisclosure();
+  const [isSyncing, setIsSyncing] = useState(false);
+  const [notionConfigured, setNotionConfigured] = useState(false);
+
+  const onClickNotion = async () => {
+    onClose();
+    notionModal.onOpen();
+    const isConfigured = await isNotionConfigured();
+    if (!isConfigured) {
+      setIsSyncing(true);
+      await runNotionSync(false);
+      setIsSyncing(false);
+      setNotionConfigured(true);
+    }
+  };
+
+  return (
+    <>
+      <Modal
+        closeButton
+        aria-labelledby="modal-title"
+        isOpen={isOpen}
+        onClose={onClose}
+        hideCloseButton={true}
+      >
+        <ModalContent className="p-4">
+          <ModalBody>
+            <div
```

Comment:
Could this be replaced with a Button? `div` with onClick handler will have no accessibility features + `bg-white`-like modifiers don't link to themes. In cases like this it's usually best to use the component library so theming/accessibility is applied by default

Reply:
Copied from tailwind template, let me see if I can get button show the same style 👍